integrationtest/ValidationProject/ValidateStrykerResults.cs

```diff
@@ -53,12 +53,32 @@ public void NetCore()
             CheckReportTestCounts(report, total: 14);
         }
 
+        [Fact]
+        [Trait("Category", "FSharp")]
```

Comment:
I don't really know how to categorize things here, I hope eventually we will run this on whatever platform and probably in the solution mode. At that point, we might need to reassess the whole test categorization.